css/syntaxes.json

```diff
@@ -27,7 +27,7 @@
   "clip-source": "<url>",
   "color": "<rgb()> | <rgba()> | <hsl()> | <hsla()> | <hex-color> | <named-color> | currentcolor | <deprecated-system-color>",
   "color-stop": "<color> <length-percentage>?",
-  "color-stop-list": "<color-stop>{2,}",
+  "color-stop-list": "<color-stop>#{2,}",
```

Comment:
The css-images-3 spec doesn't include the ``#``: https://drafts.csswg.org/css-images-3/#color-stop-syntax.

The css-images-4 spec has the ``#``, but is more complex:
https://drafts.csswg.org/css-images-4/#color-stop-syntax

I suspect the css-images-3 spec is wrong, but I'm not sure what we do in these situations.